Assets/Scripts/ThirdPersonController.cs

```diff
@@ -0,0 +1,119 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+[RequireComponent (typeof (CharacterController))]
+public class ThirdPersonController : MonoBehaviour {
+    private CharacterController _controller;
+    private Vector2 _moveInput = new Vector2 ();
+    private Vector2 _lookInput = new Vector2 ();
+    private Quaternion nextRotation;
+    private Vector3 nextPosition;
```

Comment:
I dont see these being used, so I'd say get rid of them but not a huge deal

Reply:
Old code. Removing.